packages/flutter/lib/src/rendering/stack.dart

```diff
@@ -326,14 +316,14 @@ class RenderStack extends RenderBox
     AlignmentGeometry alignment = AlignmentDirectional.topStart,
     TextDirection textDirection,
     StackFit fit = StackFit.loose,
-    Overflow overflow = Overflow.clip,
+    Clip clipBehavior = Clip.hardEdge,
   }) : assert(alignment != null),
        assert(fit != null),
-       assert(overflow != null),
+       assert(clipBehavior != null),
        _alignment = alignment,
        _textDirection = textDirection,
        _fit = fit,
-       _overflow = overflow {
```

Comment:
A slightly more gentle way to do this would be to do the following as an in-between step:

1. Don't remove `overflow` just yet.
1. Mark `overflow` here as deprecated, with something like `@Deprecated('use clipBehavior instead, see http://...')`
2. Remove the default for `overflow`, so it's `null`.
2. If `overflow` is non-null, assign to `clipBehavior` accordingly. 

Especially here in Stack, that's going to be a lot of hard breakage if you just remove `overflow`, I think.

I'm fine either way, just wanted to point this out. There are a lot of `Stack`s out there.

Reply:
Ok, in my next patch I'll only make them deprecated instead of removing them.

Reply:
https://github.com/flutter/flutter/pull/55977/commits/deaff1f1d1fbf4a7ebeae9fc9cbacfe296dee7d5 restored `overflow`, but presumably a later rebase removed it again, because `overflow` is gone in the final merged patch.

```
Compiler message:
../../Flutter/.pub-cache/hosted/pub.dartlang.org/flutter_speed_dial-1.2.5/lib/src/speed_dial.dart:249:17: Error: The getter 'Overflow' isn't defined for the class '_SpeedDialState'.
 - '_SpeedDialState' is from 'package:flutter_speed_dial/src/speed_dial.dart' ('../../Flutter/.pub-cache/hosted/pub.dartlang.org/flutter_speed_dial-1.2.5/lib/src/speed_dial.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'Overflow'.
      overflow: Overflow.visible,
                ^^^^^^^^
Exception: Build process failed
```